exodus/exodus/locale/fr/LC_MESSAGES/django.po

```diff
@@ -154,44 +155,47 @@ msgstr ""
 "à propos de vous et de vos usages.</b> Ainsi, les rapports d’εxodus vous "
 "révèlent les ingrédients du gâteau."
 
-#: templates/home.html:48
-msgid "εxodus does not decompile applications, its analysis technic is legal."
+#: exodus/templates/home.html:48
+#, fuzzy
+#| msgid ""
+#| "εxodus does not decompile applications, its analysis technic is legal."
+msgid "εxodus does not decompile applications, its analysis method is legal."
 msgstr ""
 "εxodus ne décompile pas les applications, sa technique d’analyse est légale."
```

Comment:
Do we let "technique" in French and "method" in English?
Just asking, I don't have any preference :)